src/App.svelte

```diff
@@ -20,19 +20,35 @@
   let mapState = {};
   let maps = [];
   const config = makeConfig(localConfig);
-  let settings = getInitialSettings(config);
+  let settings = getSettings(config);
   let mapboxGlAccessToken;
   let stylePresetUrls;
   configStore.set(config);
   ({ mapboxGlAccessToken, stylePresetUrls } = config);
 
+  let settingsForHash = {};
+  $: if (settings || maps || mapState) {
+    // Remove the stylesheet for a more concise hash
+    const mapsHash = JSON.parse(JSON.stringify(maps)).map(m => {
+      delete m.style;
+      return m;
+    });
+
+    settingsForHash = { ...settings, maps: mapsHash, ...mapState };
+  }
```

Comment:
Moved this into a reactive property since we were computing this a few different ways previously.